cfgov/mega_menu/management/commands/export_mega_menu.py

```diff
@@ -21,7 +21,5 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         menu = Menu.objects.get(language=options['language'])
         json.dump(
-            menu.submenus.stream_data,
-            options['filename'],
-            indent=4
+            list(menu.submenus.raw_data), options['filename'], indent=4
```

Comment:
Super minor style thing:

If the arguments are going to be on a new line from the function call, they should each have their own line, like they did prior.

Reply:
Good catch and I made this change. Usually lint picks up changes needed for styling.